ui/pages/userpreference.js

```diff
@@ -1,4 +1,4 @@
-import UserPreference from "../components/UserPreference";
+import RemoteUserPref from "../components/RemoteUserPref";
```

Comment:
This needs to go into `master`. 

// @utkarsh-pro

Reply:
@vineethvanga18 if you don't create an extension point in `master`, how will you inject this plugin's (MeshMap's) user preference component?